### PR TITLE
Add support for WiFi site surveys

### LINKED ISSUE #605: Add support for WiFi site surveys
See https://github.com/NordicSemiconductor/asset-tracker-cloud-docs/discussions/590

### PR DESCRIPTION
**Pull request by pudkrong**
Fixes #605

### CODE REVIEW

File: docs/app/WiFiGeolocation.rst [hunk 1]
```diff
@@ -0,0 +1,25 @@
+.. _app-wifigeolocation:
+
+WiFi geolocation
```
Comment (wiba-nordic) [inline docs/app/WiFiGeolocation.rst]:
```suggestion
Wi-Fi geolocation
```

File: docs/app/WiFiGeolocation.rst [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+################
+
+.. contents::
+   :local:
+   :depth: 2
+
+If wi-fi site survey geolocation (ground fix service) is implemented, the devices can be shown on the map based on the location of their nearest wi-fi access points.
```
Comment (wiba-nordic) [inline docs/app/WiFiGeolocation.rst]:
```suggestion
If Wi-Fi site survey geolocation (ground fix service) is implemented, the devices can be shown on the map based on the location of the Wi-Fi access points nearest to them.
```

File: docs/app/WiFiGeolocation.rst [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+
+Third-party location APIs
+*************************
+
+nRF Asset Tracker implements the following third-party location APIs:
+
+ - nRF Cloud Location Services
```
Comment (wiba-nordic) [inline docs/app/WiFiGeolocation.rst]:
```suggestion
* nRF Cloud Location Services
```

File: docs/app/WiFiGeolocation.rst [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+These APIs enable calculation of the rough location of a device as soon as it sends the nearest wi-fi access points information to the cloud.
```
Comment (wiba-nordic) [inline docs/app/WiFiGeolocation.rst]:
```suggestion
These APIs enable calculation of the rough location of a device as soon as it sends the information for the nearest Wi-Fi access points to the cloud.
```

File: docs/app/WiFiGeolocation.rst [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+Follow the configuration guide in the respective implementation to enable nRF Cloud's group fix API:
```
Comment (wiba-nordic) [inline docs/app/WiFiGeolocation.rst]: Should this be ground fix, not group fix?

**Review by wiba-nordic — CHANGES_REQUESTED** [review]

File: docs/app/WiFiGeolocation.rst (hunk 5 above)
Comment (peknis) [reply docs/app/WiFiGeolocation.rst]: I believe Ground Fix API is the right thing, note also the capitalization because that is how it is in the API docs.

**Review by peknis — APPROVED** [review]

File: docs/app/WiFiGeolocation.rst [hunk 6]
```diff
@@ -0,0 +1,25 @@
+.. _app-wifigeolocation:
+
+Wi-Fi geolocation
+################
```
Comment (wiba-nordic) [inline docs/app/WiFiGeolocation.rst]:
```suggestion
#################
```

**Review by wiba-nordic — CHANGES_REQUESTED** [review]
Header underlining needs to match the new added length.

**Review by coderbyheart — CHANGES_REQUESTED** [review]
Please document protocol like it was done for A-GPS and P-GPS data: https://github.com/NordicSemiconductor/asset-tracker-cloud-docs/blob/saga/docs/cloud-protocol/DeviceDataAndConfiguration.rst#a-gps-and-p-gps-data

**Comment by coderbyheart** [comment]
Reference PR: #239

**Review by wiba-nordic — APPROVED** [review]
Looks good.

**Review by coderbyheart — CHANGES_REQUESTED** [review]
Looks good, but please update the JSON example data to show the optional keys.

**Pull request merged**